### DISCUSSION TITLE
"Notes for Windows Users" is needlessly sarcastic. Install documentation in general feels a bit too sassy.

### DISCUSSION THREAD

**Discussion by Mark-LaCroix** [newren/git-filter-repo#612] [Ideas]
"For some reason lots of Windows users have a hard time modifying their PATH"
"It seems some users don't have a python3 executable"

This attitude is common, but it's usually directed at Windows itself, not Windows users. This is unproductive, and likely dissuades users from using, promoting, and teaching others how to use the tool, which creates a deadly spiral.

Other bits in the documentation have an air of _you should already know how to do this_ or _I can't believe I have to explain this to you_, like: "If for some reason downloading a single file is too much of an installation hassle for you..." This may be intended as dry wit but I suspect would turn some folks away.

An example that's potentially confusing for newer users on all platforms is the repeated use of a variation of "it's just one python file," because the release package (where beginner or otherwise timid users are likely to go for the "official" download) is just a copy of the repo, so it's actually a lot of files, and the one *.py file that _is_ in the package is a symlink to the _actual_ "one file" that the documentation refers to. This isn't well-explained, and while it's probably not _too_ tricky for most folks to tease out what to do, it's an unforced error in communication.

"Just start on step 12" is all too common in open-source, and while most users have come to expect this as a more than fair price for FOSS, and this project's docs are waaaay better than average in this regard, IMO it's not something that should be treated as a burden to whine about within the body of the documentation itself, or a place to receive catty insults based on the OS you use.

A better example can be found in the disclaimer about package managers in INSTALL.md. It explains that the list may be incomplete or out-of-date and that the maintainer isn't promising otherwise. It still has some sass, but it's genuinely helpful, contextual, and doesn't blame the user or their environment. Like, I notice winget is not listed, but that's fine because at least I didn't have to hear an insult about it 😄.

**Comment by newren** [comment]
Thanks for the feedback, and good points.  I think I've fixed these all up now.  If you spot any more, though, don't hesitate to point them out.
